pkg/config/config.go

```diff
@@ -56,6 +56,10 @@ type OFEDStateConfig struct {
 	InitContainerImage string `env:"OFED_INIT_CONTAINER_IMAGE"`
 	// UseDTK Enable use of Driver ToolKit side car to compile OFED drivers (Relevant for OpenShift only)
 	UseDTK bool `env:"USE_DTK" envDefault:"true"`
+	// UseDriversInventory enables use of the special persistent directory on the host filesystem to store OFED build
+	// artifacts. If the artifacts for the correct version and arch exist in this folder
+	// when the OFED container is started, they will be reused without driver being recompiled.
+	UseDriversInventory bool `env:"USE_DRIVERS_INVENTORY" envDefault:"true"`
```

Comment:
I dont think we need this to be derived from env var of the operator.

what i had in mind: in `state_ofed.go` just extend `mergeWithDefaultEnvs()` to always set this.

Reply:
But it's worth noting that this `USE_DRIVERS_INVENTORY` env var will not translate directly to the env var mofed container is expecting. We still need to transform it into:
```
name: NVIDIA_NIC_DRIVERS_INVENTORY_PATH
value: "/mnt/drivers-inventory"
```